lib/animas/animasSimulator.js

```diff
@@ -153,7 +153,7 @@ exports.make = function(config){
           // categorize the basal we're processing as a suspend basal if its rate
           // is zero and its start falls within the suspend event we have on deck
           var resumed = Date.parse(currStatus.time) + currStatus.duration;
-          if (resumed > Date.parse(event.time)) {
+          if (resumed >= Date.parse(event.time)) {
```

Comment:
Ah, OK. I'm glad it's not completely related because that's what was confusing me most - trying to see the connection with the other changes.